### ISSUE TITLE
Docker container for EtherCIS V1.3

### ISSUE THREAD

**Issue by PhilBarrett** [ethercis/ethercis#170]
We have a need to update our Docker containers for EtherCIS 1.3 version, as the initial deployment will be to docker based env
https://github.com/RippleOSI/ethercis-server-docker
https://github.com/RippleOSI/ethercis-database-docker
.. this update should include some test data loading for dev purposes

**Comment by chevalleyc** [comment]
I don't think this is an EtherCIS issue. Please move this issue to the relevant repository. Thanks
